### ISSUE TITLE
0.5.4 breaks nrf-hal

### ISSUE THREAD

**Issue by jonas-schievink** [rtic-rs/cortex-m-rtic#351]
Since 0.5.4 was released, we're seeing this compilation error for the [rtic-demo](https://github.com/nrf-rs/nrf-hal/blob/master/examples/rtic-demo/src/main.rs) in nrf-hal:

```
error[E0433]: failed to resolve: use of undeclared type or module `cortex_m`
  --> examples/rtic-demo/src/main.rs:22:1
   |
22 | #[app(device = crate::hal::pac)]
   | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ use of undeclared type or module `cortex_m`
```

To reproduce, check out the [nrf-hal](https://github.com/nrf-rs/nrf-hal) repo and run:

```
rm Cargo.lock
cd examples/rtic-demo
cargo build
```

**Comment by AfoHT** [comment]
That is interesting, apart from the test-suite and examples the only change is found in #338 

I found that by adding 

```
cortex-m = "0.6.3"
```
to `[dependencies]` makes it compile happily.

**Comment by jonas-schievink** [comment]
I think #338 used the wrong path for the fix, it needs to go through the `export` module https://github.com/rtic-rs/cortex-m-rtic/pull/338#discussion_r478377971

**Comment by jonas-schievink** [comment]
Fixed in 0.5.5
